From a0de4e8476641efec1373402d61d2b086db4363d Mon Sep 17 00:00:00 2001
From: AlexanderLipatov <golddragonsoul@gmail.com>
Date: Sun, 3 May 2015 14:47:45 +0700
Subject: [PATCH] fix

---
 Assets/scene_test.unity | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Assets/scene_test.unity b/Assets/scene_test.unity
index 1798fb7..8140005 100644
--- a/Assets/scene_test.unity
+++ b/Assets/scene_test.unity
@@ -169,6 +169,7 @@ Transform:
   - {fileID: 673972621}
   m_Father: {fileID: 0}
   m_RootOrder: 3
+<<<<<<< HEAD
 --- !u!1 &876569375
 GameObject:
   m_ObjectHideFlags: 0
@@ -198,6 +199,8 @@ Transform:
   - {fileID: 249223562}
   m_Father: {fileID: 0}
   m_RootOrder: 4
+=======
+>>>>>>> a32c1a13404d1675a63b1bf480143d01d46e99e8
 --- !u!1 &1221794801
 GameObject:
   m_ObjectHideFlags: 0
@@ -258,6 +261,7 @@ Transform:
   m_Children: []
   m_Father: {fileID: 0}
   m_RootOrder: 1
+<<<<<<< HEAD
 --- !u!1 &1313467747
 GameObject:
   m_ObjectHideFlags: 0
@@ -285,6 +289,8 @@ Transform:
   m_Children: []
   m_Father: {fileID: 876569376}
   m_RootOrder: 0
+=======
+>>>>>>> a32c1a13404d1675a63b1bf480143d01d46e99e8
 --- !u!1 &1619585869
 GameObject:
   m_ObjectHideFlags: 0
-- 
1.9.5.msysgit.0

